.gitlab-ci.yml

```diff
@@ -10,6 +10,11 @@ build:
   image: golang:1.11.5
   script:
     - make install
+  artifacts:
```

Comment:
Maybe this can go in another PR?

Doesn't seem directly related to the PR title

Reply:
Normally yes, but I want to have upgrade binaries automatically generated in CI. I tested it and it works...

Reply:
Okay if I keep it here for now @ethanfrey ?

Reply:
Sure. More a comment for style, and general PR flow.

But I see the urgency here and it's a super minor point. Let's get this out and merged.